### PR TITLE
Fix customer related scheduler task failure and accounts not created.

### PR DESCRIPTION
**Pull request by faerietree**
See comment of @aimeos https://github.com/aimeos/aimeos-core/pull/99#issuecomment-319919062

(sorry for the extra pull request, wanted to reuse the old one but apparently failed due to force push or because I did not reopen it before pushing ...)

### CODE REVIEW

**Comment by aimeos** [comment]
Thank you for your PR :+1:

File: lib/custom/setup/FEUsersAddSiteIdTypo3.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+	public function getPostDependencies()
+	{
+		return array('TablesCreateTypo3');
+	}
+
+
+	/**
+	 * Executes the task for MySQL databases.
+	 */
+	protected function mysql()
```
Comment (aimeos) [inline lib/custom/setup/FEUsersAddSiteIdTypo3.php]: This is deprecated. You should use `migrate()` instead:
https://aimeos.org/docs/Developers/Library/Setup_tasks/Targets#Migrate

File: lib/custom/setup/FEUsersAddSiteIdTypo3.php [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+	protected function process( array $stmts )
+	{
+		$this->msg( 'Adding siteid column to fe_users table', 0 );
+		$this->status( '' );
+		$table = 'fe_users';
+
+		$this->msg( sprintf( 'Checking table "%1$s": ', $table ), 1 );
+
+		if( $this->schema->tableExists( $table ) === true
+			&& $this->schema->columnExists( $table, 'status' ) === false )
```
Comment (aimeos) [inline lib/custom/setup/FEUsersAddSiteIdTypo3.php]: It must be `siteid` instead of `status`

**Comment by faerietree** [comment]
hm ... tested it and it seems to work. Have used `db-customer` though because `fe_users` [is not listed](https://aimeos.org/docs/Developers/Several_databases). This will definitely lead to issues if different database are used. I'm not so sure about it.

Best were to just update to TYPO3. But I'm a bit scared of all the things that'll break as it requires an OS update first, then PHP, then to fluid_styled_content, ... it will do no good to my neverending story .. eh .. list.

**Review by aimeos — APPROVED** [review]
Perfect! Thanks for the fix :-)

**Comment by faerietree** [comment]
I thank you for this epic shop engine. Reviving this arcavias was a great idea! You are one of the first companies of the open economy. Very honorable. :-) For an epic world!

**Pull request merged**